src/Cake.Issues.DupFinder/DupFinderIssuesProvider.cs

```diff
@@ -26,9 +30,135 @@ protected override IEnumerable<IIssue> InternalReadIssues()
         {
             var result = new List<IIssue>();
 
-            //// TODO Implement parsing of log file
+            var logDocument = XDocument.Parse(this.IssueProviderSettings.LogFileContent.ToStringUsingEncoding());
+            foreach (var duplicate in logDocument.Descendants("Duplicate"))
+            {
+                if (!TryGetCost(duplicate, out var cost))
+                {
+                    continue;
+                }
+
+                // Gets all fragments from a duplicate
+                var fragments = GetDuplicateFragements(duplicate);
+                if (fragments.Count < 2)
+                {
+                    continue;
+                }
+
+                foreach (var fragment in fragments)
+                {
+                    var builder = new StringBuilder();
+                    builder.AppendLine($"Possible duplicate detected (cost {cost}).");
+                    builder.AppendLine("The following fragments were found that might be duplicates:");
+                    foreach (var possibleDuplicateFragment in fragments.Where(innerDuplicate => !innerDuplicate.Equals(fragment)))
+                    {
+                        builder.Append($"`{possibleDuplicateFragment.File}` (Line {possibleDuplicateFragment.LineStart} to {possibleDuplicateFragment.LineEnd})");
```

Comment:
I would suggest to also provide message in Markdown and HTML format, which could improve output (e.g. reporting to Azure Repos pull requests uses Markdown by default if available, and generic reporting addin might use HTML)

Reply:
Do you have an example where I can base this on?
Wouldn't this be a thing you'd have to add for InspectCode aswell? I can't see it there, does it not exist for InspectCode. That's odd, because I am pretty sure I remember it not looking like a simple message when I tested it with our Azure Devops server a week ago.

Reply:
I did provide both properties now. Hope this is sufficient enough

Reply:
LGTM 👍